src/native/fs/fs_napi.cpp

```diff
@@ -1941,6 +1945,51 @@ struct DirOpen : public FSWorker
     }
 };
 
+struct TellDir : public FSWrapWorker<DirWrap>
+{
+    std::string _dir_path;
```

Comment:
I dont think storing this as a member is needed, you can just access it from _wrap->_path